packages/site/src/content/AnimatedSwitcher/index.tsx

```diff
@@ -38,4 +39,5 @@ export default {
       ),
     },
   ],
+  notes: () => <Notes />,
```

Comment:
Yup like this pattern of linking MDX files from the content page. Great work. :)